IRTweaks/IRTweaks/Modules/Misc/DeathChanceStatFix.cs

```diff
@@ -0,0 +1,73 @@
+﻿using BattleTech;
+using Harmony;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Emit;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IRTweaks.Modules.Misc
+{
+    [HarmonyPatch(typeof(Contract), "FinalizeKilledMechWarriors")]
+    static class Contract_FinalizeKilledMechWarriors
+    {
+        public static bool Prepare() => Mod.Config.Fixes.DeathChanceStat;
+
+        public static float CalcChance(float minchance, float basechance, float gutsReduction, float gutsAmount, SimGameState s, Pilot p)
+        {
+            float statmodifier = p.LethalInjuries ? GetStat(s, "LethalDeathChance") : GetStat(s, "IncapacitatedDeathChance");
+            return Math.Max(minchance, basechance - gutsReduction * gutsAmount + statmodifier);
+        }
+
+        public static float GetStat(SimGameState s, string stat)
```

Comment:
This is literally what `StatCollection.GetValue<float>()` already does - returns the value if one exists to 0.0 if one doesn't. :)

Reply:
well, didn't look into the method and just put a check in. changed it now.